aries-site/src/components/feedback/Question.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormField, TextArea, RadioButtonGroup, FileInput } from 'grommet';
+import { StarRating } from './StarRating';
+import { ThumbsRating } from './ThumbsRating';
+
+export const Question = ({ label, inputProps, kind, name }) => {
+  const accessibilityProps = {
+    id: name,
+    name,
+  };
+
+  let Input;
+  if (kind === 'star') Input = StarRating;
+  else if (kind === 'thumbs') Input = ThumbsRating;
+  else if (kind === 'textArea') Input = TextArea;
+  else if (kind === 'radioButtonGroup') Input = RadioButtonGroup;
+  else if (kind === 'fileInput') Input = FileInput;
+
+  if (kind === 'star' || kind === 'thumbs')
```

Comment:
Can we get away with a single return?

Modify the proposed `formats` object to something like:

```
const formats = {
  star: {
    render: StarRating,
    border: false,
  },
...
  textArea: {
    render: TextArea,
  }
  ...
}
```